esp-box-3/include/bsp/esp-box-3.h

```diff
@@ -20,12 +20,26 @@
 #include "esp_lvgl_port.h"
 #include "esp_codec_dev.h"
 #include "iot_button.h"
+#include "icm42670.h"
 
 #if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 0)
 #include "driver/i2s.h"
 #else
 #include "driver/i2s_std.h"
 #endif
+/**************************************************************************************************
+ *  BSP Capabilities
+ **************************************************************************************************/
+
+#define BSP_HAS_DISPLAY        1
```

Comment:
Could we rename to `BSP_CAPS_DISPLAY`? This way it will be similar to esp-idf `SOC_CAPS_...`

Reply:
Renamed.